code-challenges.test.js

```diff
@@ -14,46 +16,159 @@
-// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
-const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
-// Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+//use a describe statement:
+// use a describe method that takes an argument of an array
+// describe ("capitalized", () => {})
+//next, I use it statement which is nested inside describe block, to describe what the test is doing. 
+// it("takes in an array of words and returns an array with all the words capitalized.", () => {})
+//next, I use expect statement to call the function and matcher method .toEqual.
+//.toEqual contains the expected output of the function. It compare the actual output to the output in the test
+//to run the tesy I use yarn jest command 
+
+// describe ("capitalized", () => {
+//     it("takes in an array of words and returns an array with all the words capitalized.", () => {
+//         expect(capitalized(randomNouns1)).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+//         expect(capitalized(randomNouns2)).toEqual(["Temperature", "Database", "Chopsticks", "Mango"])
+//     })
+// })
+
+// FAIL -> ReferenceError: capitalized is not defined
 
 // b) Create the function that makes the test pass.
 
+//declare a function called capitalized that takes an array as an argument
+// use .map() method to iterate thopugh the array
+//return the array outputfrom the map action
+//access the first index of each value and convert it to uppercase using .toUpperCase() method
+//use + to concattenate the rest of the word
+
+// const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+// // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+// const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+// // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+
+// const capitalized = (array) => {
+//     return array.map((value) => {
+//         return value[0].toUpperCase() + value.substring(1)
+//     })
+
+// }
+
+//PASSED -> Test Suites: 1 passed, 1 total
+
 // --------------------3) Create a function that takes in a string and logs the index of the first vowel.
 
 // a) Create a test with expect statements for each of the variables provided.
 
+//use a describe statement:
+// use a describe method that takes an argument of a sting
+// describe ("indexOfFirstVowel", () => {})
+//next, I use it statement which is nested inside describe block, to describe what the test is doing. 
+// it("takes in a string and logs the index of the first vowel", () => {})
+//next, I use expect statement to call the function and matcher method .toEqual.
+//.toEqual contains the expected output of the function. It compare the actual output to the output in the test
+//to run the tesy I use yarn jest command 
+
+describe("indexOfFirstVowel", () => {
+    it("takes in a string and logs the index of the first vowel", () => {
```

Comment:
Your tests look really good.  I would recommend moving your test variables into the test itself just after the it statement and before the expect statement.  This helps to keep it confined to the scope of where the variables are being called.